packages/actor-sparql-serialize-stats/lib/ActorSparqlSerializeStats.ts

```diff
@@ -0,0 +1,73 @@
+import {IActorQueryOperationOutputBindings,
+  IActorQueryOperationOutputQuads} from "@comunica/bus-query-operation";
+import {ActorSparqlSerializeFixedMediaTypes, IActionSparqlSerialize,
+  IActorSparqlSerializeFixedMediaTypesArgs, IActorSparqlSerializeOutput} from "@comunica/bus-sparql-serialize";
+import {Readable} from "stream";
+
+/**
+ * Serializes SPARQL results for testing and debugging.
+ */
+export class ActorSparqlSerializeStats extends ActorSparqlSerializeFixedMediaTypes {
+
+  private startTime: [number, number] = process.hrtime();
+  private result: number = 1;
+
+  constructor(args: IActorSparqlSerializeStatsArgs) {
+    super(args);
+    if (args.delay) {
+      this.delay = args.delay;
+    }
+  }
+
+  public async testHandleChecked(action: IActionSparqlSerialize) {
+    if (['bindings', 'quads'].indexOf(action.type) < 0) {
+      throw new Error('This actor can only handle bindings streams or quad streams.');
+    }
+    return true;
+  }
+
+  public pushHeader(data: Readable) {
+    const header: string = ['Result', 'Delay (ms)', // 'Requests (#)'
```

Comment:
I guess these commented parts will go once that TODO from the PR description is resolved?